docs/book/getting-started/deploying-zenml/docker.md

```diff
@@ -1,88 +1,359 @@
+
+- **ZENML_DEFAULT_PROJECT_NAME**:
+    The name of the default project created by the server on first deployment,
+    during database initialization. Defaults to `default`.
+- **ZENML_DEFAULT_USER_NAME**:
+    The name of the default admin user account created by the server on first
+    deployment, during database initialization. Defaults to `default`.
+- **ZENML_DEFAULT_USER_PASSWORD**:
+    The password to use for the default admin user account. Defaults to an empty
+    password value, if not set.
+- **ZENML_STORE_URL**:
+    This URL should point to a SQLite database file _mounted in the container_,
+    or to a MySQL compatible database service _reachable from the container_. It
+    takes one of the forms:
+
+        sqlite:////path/to/zenml.db
     
-- ZENML_DEFAULT_PROJECT_NAME
-    The name of the default project created by the server on first deployment.
-- ZENML_DEFAULT_USER_NAME
-    The name of the default user account created by the server on first
-    deployment.
-- ZENML_DEFAULT_USER_PASSWORD
-    The password to use for the default user account.    
-
-- ZENML_STORE_TYPE
-    The type of backend store that you want to configure 
-    ZenML with. This should always be set to "sql".
-- ZENML_STORE_URL
-    This URL should point to a MysSQL compatible database. It takes the form:
+    or:
 
         mysql://username:password@host:port/database
 
-- ZENML_STORE_SSL_CA
-    The server certificate in use by the MySQL database can be passed using this
-    variable, if you are using SSL to connect to the database.
-- ZENML_STORE_SSL_CERT
-    The client certificate required to connect to the MySQL database, if you
-    are using SSL client certificates to connect to the database.
-- ZENML_STORE_SSL_KEY
-    The client certificate private key required to connect to the MySQL
-    database, if you are using SSL client certificates to connect to the
-    database.
-- ZENML_STORE_SSL_VERIFY_SERVER_CERT
-    Whether to verify SSL certificate for the MySQL server.    
+{% hint style="warning" %}
+The SQLite database backend option currently has one major limitation:
+the server will store incomplete information about pipeline runs and
+thus the pipeline runs will only be partially available in the ZenML UI. This
+comes from the requirement that the ZenML clients and orchestrators still need
+direct access to the server database to store this information.
+
+This limitation will be corrected in a future release that removes the
+dependency on the ml-metadata library for pipeline run tracking.
+
+This situation will be signalled by a warning message in the ZenML client and
+pipeline logs that looks like this:
+
+    The ZenML server is using a SQLite database at /zenml/.zenconfig/local_stores/default_zen_store/zenml.db
+    that is not available locally. Using the default local SQLite database
+    instead.
+
+In the meantime, it is recommended to configure the ZenML server to connect to a
```

Comment:
TL;DR: You can either use MySQL or SQLite, but don't use SQLite :D

Why don't we tell people to use MySQL directly?

Reply:
Nevermind, just finished reading, I agree it is pretty clean to have sensible defaults that allow you to just run the image without having to set up a MySQL DB first.

Let me change my question then: If the runs are broken, when would you use the default deployment with SQLite instead of regular `zenml up`? What's the point? If there is no use case for it, I would still propose to hide the SQLite option for now and require users to set up a MySQL with their local Docker deployments

Reply:
When we remove the need for clients to connect directly to the server's database, this SQLite scenario will become a very important easy-to-set-up ZenML server deployment option.

My (not so secret) plan is to keep the option documented and just update the docs to reflect it as the simpler option when we finally remove MLMD.